packages/web-components/fast-components/src/tabs/fast-carousel.definition.ts

```diff
@@ -0,0 +1,113 @@
+import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+
+export const fastCarouselDefinition: WebComponentDefinition = {
+    version: 1,
+    tags: [
+        {
+            name: "fast-carousel",
+            description: "The FAST Carousel element",
+            attributes: [
+                {
+                    name: "autoplay",
+                    type: DataType.boolean,
+                    description: "The autoplay attribute",
+                    default: true,
+                    required: false,
+                },
+                {
+                    name: "loop",
+                    description: "The loop attribute",
+                    type: DataType.boolean,
+                    default: true,
+                    required: false,
+                },
+                {
+                    name: "paused",
+                    description: "The paused attribute",
+                    type: DataType.boolean,
+                    default: false,
+                    required: false,
+                },
+                {
+                    name: "pattern",
+                    description:
+                        "The pattern attribute controlling if the carousel uses the Tabbed or Basic ARIA pattern",
+                    type: DataType.string,
+                    default: "tabbed",
```

Comment:
is this a set of values? if so check out the fast-anchor `appearance` for an example on how to add a set of values.